iml-wire-types/wire-types.rs

```diff
@@ -1212,6 +1212,84 @@ pub mod db {
         }
     }
 
+    /// Record from the `chroma_core_ostpool` table
+    #[derive(serde::Deserialize, serde::Serialize, Debug, Clone)]
+    pub struct OstPoolRecord {
```

Comment:
This duplicates #1364

Reply:
I wouldn't mind using your changes, but if so can you separate them out from that patch? Otherwise, let's just land this, as it's needed for this patch to function.